fontforge/parsettf.c

```diff
@@ -3999,7 +3999,7 @@ return( 0 );
 	    SCConvertToOrder2(info->chars[i]);
     }
 
-    for ( i=0; fontnames[i]!=NULL && i<1; ++i ) {
+    for ( i=0; i<1 && fontnames[i]!=NULL ; ++i ) {
```

Comment:
It's weird because up at the beginning of the function there is e.g. `for ( i=0; fontnames[i]!=NULL; ++i )` which seems to suggest that fontnames in NULL-terminated (or at least this invariant is relied upon before). 
So maybe this i<1 should just be removed...